integration/create_test.go

```diff
@@ -72,15 +69,14 @@ func TestCreateStress(t *testing.T) {
 			}
 
 			// Attempt to create a collection for Tigris with a schema.
-			// If we get an error, that's MongoDB (FerretDB ignores that argument for non-Tigris handlers),
-			// so we create collection without schema.
+			// If we get an error that support for "validator" is not implemented, that's Postgres.
+			// If we get an error that "$tigrisSchemaString" is unknown, that's MongoDB.
```

Comment:
I think this comment should come right after `// Attempt to create a collection for Tigris with a schema.`

Reply:
That comment is copy&pasted in several places, I think it should have exactly the same text to make it easier to spot that repeated problem